README.md

```diff
@@ -45,3 +45,8 @@ $ npm start
 
 The server's log level can be set in the environment or the .env file using `LOG_LEVEL=*` with one of `fatal`, `error`, `warn`, `info`, `debug`, `trace`.
 If none is given `info` is used.
+
+## Documentation
+
+This project uses [`lout`](https://github.com/hapijs/lout) to automatically generate API documention. To view the docs, start the server and visit
+[`http://localhost:2015/docs`](http://localhost:2015/docs).
```

Comment:
i feel like having the docs in the readme might be too much... i do think we could put a quick version of the available routes in their though, since they are p conventional.